resources/scripts/import-kibana-dashboards.sh

```diff
@@ -0,0 +1,22 @@
+export_dot_env() {
+  cd ../..
+  export $(cat .env | xargs)
+}
+
+import_dashboard() {
+  FILE_CONTENT=$(cat $1 | jq -r '.')
+
+  curl -X POST -u $ELASTICSEARCH_USERNAME:$ELASTICSEARCH_PASSWORD http://srv.meaningfy.ws:5601/api/saved_objects/_import?overwrite=true \
+    -H 'kbn-xsrf: true' --form file=@$1
+}
+
+import_all_dashboard() {
```

Comment:
make sure we can control the folder where the dashboards are stored, as a parameter is best